_includes/trusted.html

```diff
@@ -0,0 +1,50 @@
+<style type="text/css" scoped>
+  .trusted-section {
+    color: #eaf1f7;
+    background-color: #2c3e50;
+    padding: 4em;
+  }
+
+  .trusted-container {
+    max-width: 1600px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 1.5em;
+  }
+
+  .logo-container {
+    justify-content: space-around;
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .client-logo {
+    margin-right: 12px;
+    max-height: 68px;
+    padding: 8px;
+  }
+
+  .client-logo.svg {
+    padding: 16px 8px;
```

Comment:
This seems odd... what is the reason for this? Isn't it random that the SVG happens to not have enough padding, or? Why is `svg` the significant class name here?

Reply:
That was my dirty workaround for this specific svg that has a different padding that the remaining pngs. Proper way would be to make it in the same format (png) and with the same padding as the rest of the images here have.

Reply:
Fixed